docs/contributing/UPGRADE.md

```diff
@@ -0,0 +1,73 @@
+# Upgrading monorepo
```

Comment:
Please reconsider changing name of this file to something like "monorepo upgrade"

Reply:
We renamed the file to `upgrade-monorepo.md`